src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -289,6 +304,26 @@ public struct S
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void ReadOnlyMethod_CallBaseMethod()
+        {
+            var csharp = @"
+public struct S
+{
+    readonly void M()
+    {
+        // no warnings for calls to base members
+        GetType();
+        ToString();
+        GetHashCode();
+        Equals(null);
+    }
+}
+";
+            var comp = CreateCompilation(csharp);
+            comp.VerifyDiagnostics();
+        }
```

Comment:
>} [](start = 8, length = 1)

Please test with overridden `ToString()`, `GetHashCode()`, `Equals()`, with calls through `this` and `base`.

Reply:
Done